### DISCUSSION TITLE
Atmospheric

### DISCUSSION THREAD

**Discussion by supermassimo** [DynamicTreesTeam/DynamicTrees#906] [Compatibility Addon Suggestions]
https://www.curseforge.com/minecraft/mc-mods/atmospheric
+ Tens of biomes
+ Tens of trees

**Comment by TheFastHero** [comment]
1.20.1 also!

**Comment by szunakabeats** [comment]
is this being worked on ? :)

**Comment by cagemop** [comment]
1.21.1 also!

**Comment by szunakabeats** [comment]
please bring this to life

**Comment by Greenblyat** [comment]
I.20.1 Forge for me, would love this
